java/runtime/src/main/java/io/ray/runtime/object/ObjectSerializer.java

```diff
@@ -121,6 +124,30 @@ public static Object deserialize(
     }
   }
 
+  public static NativeRayObject serializeArrow(Object object) {
+    if (object instanceof NativeRayObject) {
```

Comment:
It seems that `serializeArrow` and `serialize` are almost the same. Could you avoid duplicated code?